content/en/ecosystem/integrations.md

```diff
@@ -1,12 +1,21 @@
 ---
 title: Integrations
-description: OpenTelemetry integrations with other open-source projects
+description:
+  Libraries, services and apps with first-party support of OpenTelemetry.
 aliases: [/integrations]
-cSpell:ignore: buildx containerd
 ---
 
-A non-exhaustive list of libraries, services and apps that integrate
-OpenTelemetry **natively** to emit traces, metrics and logs.
+OpenTelemetry's mission is
+[to enable effective observability by making high-quality, portable telemetry ubiquitous](/community/mission/).
+This means, that we not only want you to get observability for the applications
+you develop, but that observability should be built in to all the software you
```

Comment:
… software, such as? do we want to call out APIs, cloud providers, frameworks?

Reply:
I'd like to keep it rather generic at this place and also not write "libraries, services, and apps" once again.